spec/index.html

```diff
@@ -644,6 +654,15 @@ <h3>IRIs</h3>
         if a <a data-cite="rfc3986#section-5.1">base IRI
         can be established</a> [[RFC3986]].</p>
 
+      <p><strong>URI Schemes:</strong>
+        Implementations are encouraged to enforce the scheme-specific rules of
+        the common schemes, such as the
+        <a data-cite="RFC7230#section-2.7.1">scheme rules for HTTP/HTTPS</a> and the
+        <a data-cite="did-core#identifier">DID syntax</a>.
+        Implementations ignore URI scheme rules
```

Comment:
```suggestion
        Implementations are expected to ignore URI scheme rules
```

Reply:
I don't think that is necessary. "expected to" sounds like a choice.
(NB This is a note and does not contain 2119 language).

Reply:
I'll go along if no-one else in the group sees this as I do.